toolchain/check/impl_lookup.h

```diff
@@ -49,9 +49,10 @@ auto LookupMatchesImpl(Context& context, SemIR::LocId loc_id,
 
 // The result of EvalLookupSingleImplWitness(). It can be one of:
 // - No value. Lookup failed to find an impl declaration.
-// - A concrete value. Lookup found a concrete impl declaration that can be
+// - A constant. Lookup found a concrete impl declaration that can be
 //   used definitively.
-// - A symbolic value. Lookup found an impl but it is not returned since lookup
+// - A symbolic value. Lookup found an impl but it is not returned since
+// lookup
```

Comment:
Please undo this change. If wrapping is needed, the extra words should be combined into the next line.
```suggestion
// - A symbolic value. Lookup found an impl but it is not returned since lookup
```